### ISSUE TITLE
Support multiple logs from a single issuer for incident management

### ISSUE THREAD

**Issue by bwesterb** [ietf-plants-wg/merkle-tree-certs#26]
We should spend a few words on what to do with bit flips.

**Comment by bwesterb** [comment]
I do not think we should cover all this in the draft, but it's a good exercise.

1. **Bitflip in assertion** at CA. Say an assertion is published for `gnogle.com` instead of `google.com`. This should be counted as a misissuance, and detecting these is one of the reasons for CT. I'd say it's out of scope to decide what should happen next: that's a policy matter of the root programs/RPs.
2. **Bitflip when computing authentication path** at CA. The subscriber will get a certificate that simply doesn't validate. The ACME client can check the certificate before installing it. This will be caught early on without lasting impact.
3. **Bitflip when signing a Window** at CA. This one is trickier. On the one hand, when checking the signed window, the TS see that the batch tree heads do not match. They could simply ignore that particular window. However, the situation would be similar to when a CA tries to split the world view. I guess we will need to be tough on this one too. It's not impossible to avoid for the CA: before publishing the signed window, the CA should check whether it verifies to detect any bit flip.
6. **Bitflip when serving assertions** at CA. This is related to the previous. A bit flip might make the CA serve the wrong assertion. I think we can be more lenient here. If the TS fetches the assertions a second time, it'll probably get the correct values and the window signature will validate.
7. **Bitflip when serving a Window** at CA. If the bit flip is in the signature, we can be lenient. It's important that the signed message is explicit: if we would not have included the tree heads in the signed window, then we couldn't be lenient here. If the bit flip is in one of the tree head, we can also be lenient: the signature will not validate and thus we know it's not part of a split world view attack.

**Comment by davidben** [comment]
Post-Photosynthesis triage: Leaving this open to track adding some text about how to recover from policy violations

**Comment by davidben** [comment]
(Still need to write it up, but some of us have been noodling over a "log generation" design to allow CAs to non-destructively cycle to a new log when an existing log goes wrong.)

**Comment by davidben** [comment]
Assigning this to myself. I'll put together a PR for log generations.

**Comment by davidben** [comment]
(Actually @nharper is probably going to do it, but I'm not able to assign it to him on GitHub for some reason.)

**Comment by nharper** [comment]
(I'm not able to assign it to myself, but I'm going to write up log generations aka "the Issuer identifies an entity that can run multiple logs and there's a log number in the upper bits of the serial".)

**Comment by davidben** [comment]
This is done now, forgot to close it.
